zone/entity.cpp

```diff
@@ -505,6 +505,22 @@ void EntityList::MobProcess()
 					zone->GetSecondsBeforeIdle(),
 					zone->GetSecondsBeforeIdle() != 1 ? "s" : ""
 				);
+
+				TraderRepository::DeleteWhere(
+					database,
+					fmt::format(
+						"`char_zone_id` = '{}' AND `char_zone_instance_id` = '{}'",
+						zone->GetZoneID(),
+						zone->GetInstanceID())
+				);
+				BuyerRepository::DeleteBuyers(database, zone->GetZoneID(), zone->GetInstanceID());
+
+				LogTradingDetail(
```

Comment:
Can we make this action a function that describes what its doing, not really sure at a first glance

Reply:
Updated.